_posts/2019-07-29-vim-basics.markdown

```diff
@@ -0,0 +1,111 @@
+---
+layout: post
+title: vim Basics
+date: 2019-07-29 22:28:02
+categories:
+author: bronzdoc
+---
+
+# VIM basics
+
+I would not bore you by telling you the history of vim, and the differences between vi and vim and all of its philosophy...
```

Comment:
@bronzdoc I think you should capitalize the `V` every time you say vim
```suggestion
I would not bore you by telling you the history of Vim, and the differences between vi and Vim and all of its philosophy...
```

Reply:
@bronzdoc Also, add a link to the official page or github repo the first time you mention it